src/app/containers/Bulletin/__snapshots__/index.test.jsx.snap

```diff
@@ -968,7 +968,7 @@ exports[`Bulletin Container snapshots should render a Radio bulletin correctly 1
         >
           <span
             class="c6"
-            lang="en-GB"
+            lang="ig"
```

Comment:
If this content is in the same language as the page it doesn't need a lang attribute

Reply:
If this is the case, I think that we just have to pass the lang attribute as `en-GB` when the translations of the `ctaText` for the service is either `Watch` or `Listen`.